contribute/documentation/README.md

```diff
@@ -1,7 +1,14 @@
 # Contribute to our documentation
 
-Welcome. We're glad you're here to help make our technical documentation even better.
+Welcome! We're glad you're here.
 
-For our style guide and writing guidelines, see [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/).
+[Writers' Toolkit](https://grafana.com/docs/writers-toolkit/) is an open-source project intended to empower both internal and external contributors to help make our technical documentation even better.  
 
-If you're interested in contributing to the Writers' Toolkit, refer to the [Writers' Toolkit](https://github.com/grafana/writers-toolkit) repository.
+Go to [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/) for the following resources:
+
+- **Contribute** - Learn how can you contribute to Grafana Labs documentation.
+- **Structure** - Learn how to organize and structure our documentation.
+- **Writing and style guide** - Guidelines that cover aspects of content such as style, tooling, workflows, markdown syntax, and more.
+- **Build and review guide** - Instructions for testing, linting, validating, and backporting our documentation.
```

Comment:
Would these benefit from being links?

Reply:
i'd rather not add links because of added maintenance burden and low user benefit.